src/modules/evbc/simulator-sample-data/update-simulator-data.js

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env node
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2024 Pionix GmbH and Contributors to EVerest
+import WebSocket from 'ws';
+import fs from 'fs';
+import path, {dirname} from 'path';
+import process from 'process';
+import {fileURLToPath} from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+// Default URL of the EVerest WebSocket instance
+const defaultUrl = 'ws://localhost:8849';
+const args = process.argv.slice(2);
+const url = args.find(arg => arg.startsWith('--url='))?.split('=')[1] || defaultUrl;
```

Comment:
If I see it correctly, currently an URL like `domain.com/page?arg=value` would be read as `domain.com/page?arg` due to the `.split('=')[1]`? Shouldn't be much of a problem, but could be solved by joining all splits 1..n